tools/topology/topology1/CMakeLists.txt

```diff
@@ -236,9 +236,8 @@ set(TPLGS
         "sof-mt8195-mt6359-rt1019-rt5682\;sof-mt8195-mt6359-max98390-rt5682"
 	"sof-mt8195-mt6359-rt1019-rt5682\;sof-mt8195-mt6359-max98390-rt5682-rtnr\;-DCHANNELS=2\;-DRTNR"
 	"sof-mt8195-mt6359-rt1019-rt5682\;sof-mt8195-mt6359-max98390-rt5682-google-aec-rtnr\;-DGOOGLE_RTC_AUDIO_PROCESSING\;-DCHANNELS=2\;-DRTNR"
```

Comment:
can you provide more context to this change. it looks like you are removing 2 topologies and maybe replacing them with another one?

Reply:
We use `sof-mt8186` only on our 8186 demo board,  `sof-mt8186-mt6366-rt1019-rt5682s` and `sof-mt8186-mt6366-da7219-max98357` will not be used anymore.

Reply:
@chunxu-li can you please add this info in the git commit message. having it on github PR description would be of no use for people looking at git history.

![image](https://user-images.githubusercontent.com/62316/186644999-5804ba1e-93a5-4b30-b7e7-40b558602b8b.png)

Reply:
done